pkg/dependencies/factory.go

```diff
@@ -31,6 +32,7 @@ type Dependencies struct {
 	Kubectl                   *executables.Kubectl
 	Govc                      *executables.Govc
 	Tink                      *executables.Tink
+	PBNJ                      *pbnj.Pbnj
```

Comment:
nit: I see `Pbnj` and `PBNJ` throughout the PR. Let's stick to CamelCase for consistency
```suggestion
	Pbnj                      *pbnj.Pbnj
```